Makefile

```diff
@@ -524,6 +524,24 @@ quick-build:
 	$(MAKE) retina-image PLATFORM=linux/amd64 BUILDX_ACTION=--push
 	$(MAKE) retina-operator-image PLATFORM=linux/amd64 BUILDX_ACTION=--push
 
+.PHONY: quick-build-devcontainer
+quick-build-devcontainer: # buildx push doesnt work for some reason in kind
```

Comment:
instead of `*-devcontainer` targets, maybe we could have a `REGISTRY=local` flag or something like that?

Reply:
i had these separate to not affect existing commands, and some of the steps here are pretty much tied to devcontainer (like the registry:port) 
I can update these though if you think it is better

Reply:
i prefer some kind of flag/option rather than duplicate targets if everything else is identical, but i guess that can be less discoverable also. no strong opinion, your call

Reply:
My preference is for dev-related things to have a fast alias and prod-build things to have more flags and knobs. Yes, there's always shell aliases, but it is nice when the standard dev experience doesn't require creating aliases to be productive. It's easy to put whatever flags / tuning are needed in CI, so I'm not as bothered by that (and +1 to the discoverability problem for devex).